src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -64,22 +65,30 @@ public sealed class PolicyContract : NativeContract
         /// </summary>
         public const uint MaxStoragePrice = 10000000;
 
+        /// <summary>
+        /// The maximum MaxValidUntilBlockIncrement value that the committee can set.
+        /// </summary>
+        public const uint MaxMaxValidUntilBlockIncrement = 100500; // TODO
```

Comment:
We need to decide about maximum policy constraint for this value.

@roman-khimov

Reply:
Is this important that will block the review?

Reply:
It's not important for sure, you may review. It's just a reminder.

Reply:
It was initially set to be a day of 15s blocks. If we're talking about _maximum_ values here we should think of a day of 1s blocks at least (86400). Faster networks are likely to use lower values anyway, but at least it should be possible.

Reply:
86400 is applied, it seems reasonable.